.kitchen.yml

```diff
@@ -6,3 +6,7 @@ suites:
   - name: default
     run_list:
       - recipe[proxytrack::default]
+  - name: proxytrack
+    run_list:
+      - recipe[proxytrack::default]
+      - recipe[proxytrack-test::resource_test]
```

Comment:
Since this is primarily a resource cookbook, let's just test this all in one suite.